src/client/ns_turn_ioaddr.c

```diff
@@ -29,7 +29,9 @@
  */
 
 #include "ns_turn_ioaddr.h"
+#if defined (__linux__)
```

Comment:
Needs ` || defined (__APPLE__)` on macOS
Otherwise introduces a lot of compilation errors

Reply:
Can you add CI for mac os?